### PR TITLE
show all links for multivariant editors

### PR DESCRIPTION
**Pull request by pelikhan**
Display links to each compile service in multi-variant editors.

image

### CODE REVIEW

**Comment by pelikhan** [comment]
Worth adding to stable6.0 so that we can easily find the CODAL repoes.

File: webapp/src/dialogs.tsx [hunk 1]
```diff
@@ -68,7 +74,7 @@ export function showAboutDialogAsync(projectView: pxt.editor.IProjectView) {
 }
 
 
-function renderCompileLink(cs: pxt.TargetCompileService) {
+function renderCompileLink(variantName: string, cs: pxt.TargetCompileService) {
```
Comment (jwunderl) [inline webapp/src/dialogs.tsx:77]: Is `variantName` intended to be used anywhere in here? Seems fine as it's taking the info from the cs

**Review by jwunderl — APPROVED** [review]

File: webapp/src/dialogs.tsx (hunk 1 above)
Comment (pelikhan) [reply webapp/src/dialogs.tsx:77]: right, i taught i would never but eventually i didn't. The repo description is actually enough.

**Review by mmoskal — APPROVED** [review]

**Review by livcheerful — APPROVED** [review]

**Comment by abchatra** [comment]
I will not port this. At this point only absolutely necessary fixes will be ported. I will take this post release as a patch.

**Pull request merged**